src/app/views/cheat-sheets/vehicle-fuel-bonus/vehicle-fuel-bonus.component.html

```diff
@@ -2,13 +2,13 @@
 
     <ul>
         <li>
-            <a href="https://wiki.factorio.com/Fuel" target="_blank" rel="noopener">Fuel</a> affects train max speed and
-            how fast trains accelerate.</li>
+            Use better <a href="https://wiki.factorio.com/Fuel" target="_blank" rel="noopener">Fuel</a> to increase the maximum speed and
```

Comment:
If using this wording the next bullet also needs to change
"Research braking force to improve how fast the trains can slow down" or something similar.
I dont think this is needed.

Reply:
This change is also dropped.